### ISSUE TITLE
Unknown string value class: class com.jetbrains.python.psi.impl.PyNumericLiteralExpressionImpl

### ISSUE THREAD

**Issue by rgarzonsisinfo** [wt-io-it/odoo-pycharm-plugin#59]
Plugin Version: 0.6.0
IntelliJ Version: PyCharm 2022.2.2
IntelliJ Build: PY-222.4167.33

<details><summary>java.lang.Throwable: Unknown string value class: class com.jetbrains.python.psi.impl.PyNumericLiteralExpressionImpl</summary>

```
java.lang.Throwable: Unknown string value class: class com.jetbrains.python.psi.impl.PyNumericLiteralExpressionImpl
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:184)
	at at.wtioit.intellij.plugins.odoo.PsiElementsUtil.getStringValueForValueChild(PsiElementsUtil.java:178)
	at at.wtioit.intellij.plugins.odoo.models.OdooModelUtil.detectName(OdooModelUtil.java:37)
	at at.wtioit.intellij.plugins.odoo.models.OdooModelUtil.detectName(OdooModelUtil.java:23)
	at at.wtioit.intellij.plugins.odoo.models.index.OdooModelDefinition.<init>(OdooModelDefinition.java:19)
	at at.wtioit.intellij.plugins.odoo.models.index.OdooModelFileIndex$OdooModelFileIndexer.lambda$mapWatched$0(OdooModelFileIndex.java:120)
	at at.wtioit.intellij.plugins.odoo.PsiElementsUtil.walkTree(PsiElementsUtil.java:94)
	at at.wtioit.intellij.plugins.odoo.PsiElementsUtil.walkTree(PsiElementsUtil.java:82)
	at at.wtioit.intellij.plugins.odoo.PsiElementsUtil.walkTree(PsiElementsUtil.java:72)
	at at.wtioit.intellij.plugins.odoo.models.index.OdooModelFileIndex$OdooModelFileIndexer.mapWatched(OdooModelFileIndex.java:117)
	at at.wtioit.intellij.plugins.odoo.index.OdooDataIndexer.lambda$map$0(OdooDataIndexer.java:13)
	at at.wtioit.intellij.plugins.odoo.index.IndexWatcher.runIndexJob(IndexWatcher.java:25)
	at at.wtioit.intellij.plugins.odoo.index.OdooDataIndexer.map(OdooDataIndexer.java:13)
	at at.wtioit.intellij.plugins.odoo.index.OdooDataIndexer.map(OdooDataIndexer.java:9)
	at at.wtioit.intellij.plugins.odoo.index.OdooIndexExtension.map(OdooIndexExtension.java:26)
	at at.wtioit.intellij.plugins.odoo.index.OdooIndexer.lambda$map$0(OdooIndexer.java:19)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	...
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
	at at.wtioit.intellij.plugins.odoo.index.OdooIndexer.map(OdooIndexer.java:21)
	at at.wtioit.intellij.plugins.odoo.index.OdooIndexer.map(OdooIndexer.java:11)
	at com.intellij.util.indexing.impl.MapReduceIndex.mapByIndexer(MapReduceIndex.java:308)
	...
	at com.intellij.util.indexing.FileBasedIndexEx.getValues(FileBasedIndexEx.java:136)
	at at.wtioit.intellij.plugins.odoo.index.OdooIndex.getAllKeys(OdooIndex.java:62)
	at at.wtioit.intellij.plugins.odoo.models.impl.OdooModelServiceImpl.hasModel(OdooModelServiceImpl.java:76)
	at at.wtioit.intellij.plugins.odoo.models.inspection.MissingModelDefinitionInspection$1.visitElement(MissingModelDefinitionInspection.java:64)
	at com.intellij.psi.impl.source.tree.LeafPsiElement.accept(LeafPsiElement.java:208)
	...
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
```
</details>

**Comment by bigbear3001** [comment]
thanks @rgarzonsisinfo for reporting this, it's fixed in 0.6.2 (pending jetbrains approval). if you are eager to test it you can download the jar file from https://github.com/wt-io-it/odoo-pycharm-plugin/tree/v0.6.2

**Comment by bigbear3001** [comment]
0.6.2 has been approved and is now available in the marketplace
